src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolStcicJudicialRoleMappingTest.java

```diff
@@ -59,7 +59,9 @@ static void assertCommonRoleAssignmentAttributes(RoleAssignment r, String office
         assertEquals(TestDataBuilder.id_2, r.getActorId());
         assertEquals(RoleType.ORGANISATION, r.getRoleType());
         assertEquals(RoleCategory.JUDICIAL, r.getRoleCategory());
-        assertNull(r.getAttributes().get("bookable"));
+        if (!r.getRoleName().equals("fee-paid-judge")) {
```

Comment:
Can you add an else to confirm flag is set for `"fee-paid-judge"`?

Reply:
addressed in commit: [3d58026](https://github.com/hmcts/am-org-role-mapping-service/pull/1529/commits/3d58026b22b640a3927688b9dd6c7951466c1f18)